README.md

```diff
@@ -0,0 +1,39 @@
+# Re-usable workflows for Ledger embedded applications
+
+This project contains several reusable Github workflows meant to be included in Ledger embedded applications repositories.
+
+Several of them are mandatory in order to ensure a level of quality and standardisation among all applications.
+The mandatory workflows must pass for an application to be deployed on the Ledger store. 
+
+Others are optional and are provided here to help developers create their own application for Ledger devices.
+
+## Description of available workflows
+
+- `app_compliance.yml`\
+	This workflow is mandatory, itwill call several child reusable workflows.
+	- `_check_icons.yml`\
+		will ensure that the icons and glyphs used in your app will be displayable on the device.
+	- `_check_makefile.yml`\
+		will ensure that your Makefile is up to date and compatible with our deployment scripts.
+	- `_check_readme.yml`\
+		will ensure that your README is up to date.
+	- `_check_clang_static_analyzer.yml`\
+		will ensure that your application can compile and will perform quality checks.
+
+- `build.yml` \
+This workflow will perform a build and upload the artifact containing the compiled application.
+This workflow is optional and is meant to help developers have their own CI.
```

Comment:
I don't get why this should be optional.

Reply:
The `call_clang_static_analyzer` flow will build the application and enforce it's good compilation without warnings, having the simple build flow too would be redundant IMO.
This reusable flow is meant as an optional helper for the test CI

Reply:
After discussion on slack I've added it